<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecElaborationPage</class>
 <widget class="QWidget" name="SpecElaborationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Application Specification</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="initialInputPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="instructionLabel">
         <property name="text">
          <string>Please provide the initial specification for your target application.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="inputTabWidget">
         <widget class="QWidget" name="uploadTab">
          <attribute name="title">
           <string>Upload Specification Document(s)</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="uploadPathLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseFilesButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="processFilesButton">
             <property name="text">
              <string>Process Uploaded Document(s)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="textInputTab">
          <attribute name="title">
           <string>Enter Brief Description</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPlainTextEdit" name="briefDescriptionTextEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="processTextButton">
             <property name="text">
              <string>Process Brief Description</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="complexityReviewPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="reviewHeaderLabel">
         <property name="text">
          <string>Project Complexity &amp; Risk Assessment</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="analysisResultTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="confirmAnalysisButton">
         <property name="text">
          <string>Confirm Assessment &amp; Proceed to Final Review</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="finalReviewPage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="finalReviewHeaderLabel">
         <property name="text">
          <string>Final Specification Review</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="specDraftLabel">
             <property name="text">
              <string>Specification Draft (Editable)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="specDraftTextEdit"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="aiIssuesLabel">
             <property name="text">
              <string>AI Analysis &amp; Potential Issues</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="aiIssuesTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="feedbackLabel">
         <property name="text">
          <string>Your Feedback &amp; Clarifications (Optional):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="feedbackTextEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="submitFeedbackButton">
           <property name="text">
            <string>Submit Feedback &amp; Refine Draft</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="approveSpecButton">
           <property name="text">
            <string>Approve Specification &amp; Proceed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>